src/bentoml/_internal/models/model.py

```diff
@@ -135,18 +145,21 @@ def __hash__(self) -> int:
         return hash(self._tag)
 
     @classmethod
+    @inject
     def create(
         cls,
         name: Tag | str,
         *,
         module: str,
         api_version: str,
         signatures: ModelSignaturesType,
+        context: ModelContext,
         labels: dict[str, str] | None = None,
         options: ModelOptions | None = None,
         custom_objects: dict[str, t.Any] | None = None,
         metadata: dict[str, t.Any] | None = None,
-        context: ModelContext,
+        use_tempfs: bool = True,
```

Comment:
Honestly, instead of using the tempfs can we do this in a non-memory tmp location? The original reason we use a move at the end was to make the process atomic, and really we should never have used a memory fs for this.

We also don't have to provide this option then.

Reply:
maybe a NamedTemporaryDir?